examples/config-without-defaults/src/test/java/gov/nasa/jpl/aerie/configwithoutdefaults/ConfigurationTest.java

```diff
@@ -0,0 +1,50 @@
+package gov.nasa.jpl.aerie.configwithoutdefaults;
+
+import java.util.Map;
+
+import gov.nasa.jpl.aerie.configwithoutdefaults.generated.ActivityTypes;
+import gov.nasa.jpl.aerie.configwithoutdefaults.generated.ConfigurationMapper;
+import gov.nasa.jpl.aerie.merlin.framework.junit.MerlinTestContext;
+import gov.nasa.jpl.aerie.merlin.framework.junit.MerlinExtension;
+import gov.nasa.jpl.aerie.merlin.protocol.model.ConfigurationType;
+import gov.nasa.jpl.aerie.merlin.protocol.types.SerializedValue;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@TestInstance(Lifecycle.PER_CLASS)
+public final class ConfigurationTest {
+
+  @RegisterExtension
+  public static final MerlinExtension<Mission> ext = new MerlinExtension<>();
+
+  private static final Integer a = 42;
+  private static final Double b = 3.14;
+  private static final String c = "JPL";
+
+  private final Mission model;
+
+  public ConfigurationTest(final MerlinTestContext<Mission> ctx)
+  throws ConfigurationType.UnconstructableConfigurationException
+  {
+    // Rely on config. defaults by instantiating config. with empty argument map
```

Comment:
This comment is out of date, threw me for a loop for a minute

Reply:
Ahh, I see what you're thinking. This is actually still accurate since an empty map still means "rely on the config. to provide defaults". The only difference here is that our users no longer have to write their own config. mapper that accepts an empty map, that mapper is now auto-generated. If the config. has no defaults (all args. are required by the planner) then passing an empty map like this results in a `UnconstructableConfigurationException`.